lib/services/ios-project-service.ts

```diff
@@ -655,12 +711,16 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 	}
 
 	public async prepareProject(projectData: IProjectData, platformSpecificData: IPlatformSpecificData): Promise<void> {
-		let provision = platformSpecificData && platformSpecificData.provision;
+		const projectRoot = path.join(projectData.platformsDir, "ios");
```

Comment:
I don't believe there is a lowercase `"ios"` global constant anywhere. Feel free to use `IOS_PLATFORM_NAME` from this file - it seems reasonable